core/domain/contribution_stats_services.py

```diff
@@ -252,16 +253,20 @@ def get_translation_submitter_total_stats(
             result.
         topic_ids: List[str]|None. List of topic ID(s) to fetch
             contributor stats for.
-        max_days_since_last_activity: int. To get number of users
-            who are active in max_days_since_last_activity.
+        max_days_since_start_date_for_last_activity: int. Number of days before
+            today to start including users with contributions.
```

Comment:
I can not understand these docstrings. Does this refer to the number of days before today when users started contributing?? Ditto elsewhere.

Reply:
It is the number of days before today. We will start including user into the response, who have a contribution before  "max_days_since_start_date_for_last_activity" days from today.